Gemfile

```diff
@@ -12,3 +12,4 @@ gem "pry"
 gem "colorize"
 gem "nokogiri"
 gem "open-uri-cached"
+gem 'scraped_page_archive', github: 'everypolitician/scraped_page_archive'
```

Comment:
When you add things in this format, you should make sure that it has the secure bundler setup:

```ruby
git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
```